components/Stats.vue

```diff
@@ -313,23 +300,22 @@ export default {
     DropDown
   },
   data() {
-    const currentCounty = 'San Francisco County'
-    const CountyData = formatCountyData(Data)
-    const ConsolidatedData = consolidateAllData(Data)
-    const countyNames = Object.keys(Data)
+    for (const countyId in DataVTwo) {
+      const { name } = DataVTwo[countyId]
+      if (!name.endsWith('County')) {
+        DataVTwo[countyId].name = name + ' County'
+      }
+    }
 
-    const totalCases =
-      ConsolidatedData.cases[ConsolidatedData.cases.length - 1].cumulative
-    const totalDeaths =
-      ConsolidatedData.cases[ConsolidatedData.cases.length - 1].deathCumulative
+    const currentCounty = 'san_francisco'
+    const CountyData = formatCountyData(DataVTwo)
```

Comment:
If possible, we want to update "forCountyDataVTwo" and use it here so that formatCountyData can be obsolete.